backend/graphql/types/courseType.ts

```diff
@@ -1,13 +1,33 @@
 import { gql } from "apollo-server-express";
 
 const courseType = gql`
+  type ModuleDTO {
+    id: ID!
+    title: String!
+    description: String
+    image: String
+    previewImage: String
+    published: Boolean!
+    lessons: [String]
+  }
+
+  input ModuleInputDTO {
+    id: ID
+    title: String!
+    description: String
+    image: String
+    previewImage: String
+    published: Boolean!
+    lessons: [String]
```

Comment:
Think lessons should always exist, but it can be empty. Also, should always be of type string:
```suggestion
    lessons: [String!]!
```

Reply:
also, nitpick: `ModuleInputDTO` -> `ModuleRequestDTO`